src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,30 @@
+package lotto.domain;
+
+import java.util.List;
+
+public class Lotto {
+
+    private final List<Integer> numbers;
+
+    public Lotto(List<Integer> numbers) {
+        if(numbers.size() != 6) {
+            throw new IllegalArgumentException("로또번호는 6개여야 합니다.");
+        }
```

Comment:
```suggestion
        if (numbers.size() != 6) {
            throw new IllegalArgumentException("로또번호는 6개여야 합니다.");
        }
```
6 이라는 숫자도 매직넘버에 해당됩니다.

그리고 로또에 중복된 번호가 들어올 수 있을 것 같아요
지금 구조라면 1을 6개 가진 로또가 생성될 수 있지 않을까요?

Reply:
관련해서 테스트 코드도 작성되면 좋겠네요!